test/unit_tests/braket/aws/test_aws_quantum_task_batch.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2019-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import random
+import uuid
+from unittest.mock import Mock, PropertyMock, patch
+
+import pytest
+from common_test_utils import MockS3
+
+from braket.aws import AwsQuantumTaskBatch, AwsSession
+from braket.circuits import Circuit
+from braket.tasks import GateModelQuantumTaskResult
+
+S3_TARGET = AwsSession.S3DestinationFolder("foo", "bar")
+
+
+@patch("braket.aws.aws_quantum_task.AwsQuantumTask.create")
```

Comment:
Can we have a test for larger batch sizes (e.g. 50), for when we actually have to wait for tasks to complete?

Reply:
This is actually already done in the batch integ test, where `max_parallel` is dropped to 5 and there are 15 tasks in the batch